src/utils.js

```diff
@@ -358,9 +360,10 @@ const utils = {
           body,
         };
       } finally {
-        context.disconnectAll();
+        esiContext.disconnectAll();
```

Comment:
I don't think this is correct. you disconnect all connections, but only create 1 context. wouldn't the request of an include to an include be closed?

I think the current implementation is more robust.

Reply:
good point.